Terminal.Gui/Core/PosDim.cs

```diff
@@ -85,7 +85,7 @@ internal override int Anchor (int width)
 
 			public override string ToString ()
 			{
-				return $"Factor({factor})";
```

Comment:
Well I don't mind but did you notice that that reading the logs you won't know if it's a `PosFactor` or a `DimFactor`. How do you distinguish them? I'm only preventing this situations happens, because you regrets, then you'll do all the tons of code changes.

Reply:
I think the context of Pos v Dim is pretty obvious.

Reply:
I already did all the tons of code changes once ;-)

Reply:
Ok, I already understand you want have only the commons names of both, but why you didn't do the same rule to the `DimView`? Did you forgot or other reason? Please confirm if you really want I revert the changes and if you also want I change the `DimView` to `View` like you did to the `PosView`. Thanks.

Reply:
Yep, missing `DimView` was an oversight. Please do the change so they are consistent.

Reply:
Done.